exchanges/stream/websocket_test.go

```diff
@@ -355,7 +503,14 @@ func TestConnectionMonitorNoConnection(t *testing.T) {
 	ws.ShutdownC = make(chan struct{}, 1)
 	ws.exchangeName = "hello"
 	ws.trafficTimeout = 1
-	go ws.connectionMonitor()
+	ws.verbose = true
+	ws.Wg = &sync.WaitGroup{}
+	ws.connectionMonitor()
+	ws.connectionMonitor() // This one should exit
```

Comment:
You could add a check to `IsConnectionMonitorRunning`now that we can't rely on a returned error